example/index.html

```diff
@@ -302,10 +288,10 @@
 					<h2>Themes</h2>
 					<p>
 						Reveal.js comes with a few themes built in: <br>
-            <a href="../common/?theme=gdicool#/themes">GDI Cool</a> -
-            <a href="../common/?theme=gdilight#/themes">GDI Light</a> -
-            <a href="../common/?theme=gdisunny#/themes">GDI Sunny</a> -
-            <a href="../common/?theme=gdidefault#/themes">Default</a>
+            <a href="/?theme=gdicool#/themes">GDI Cool</a> -
```

Comment:
I just realized that these links are probably broken because of how the original repository was structured. Would you mind helping us link to the common link theme directory?